### PR TITLE
Add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by jugosack**
Dear Reviewer,

Please find activities towards the general requirements:
-  there are no linter errors](https://github.com/microverseinc/linters-config).
- It is used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
- Work is documented [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

HTML/CSS requirements
- Followed is yours list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Project requirements
IMPORTANT NOTE: For each Microverse project, you need to create a separate pull request. [Read how to work with multiple pull requests in your one repository in the most efficient way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/multiple_pull_requests.md).

The section is build for this milestone:
- the contact form in the footer.
- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Collect data by using the [Formspree service](https://formspree.io/html).
- In order to lay out the elements on the page you should use Flexbox.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -372,8 +372,19 @@ <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">
...
   </div>
- 
+ <footer class="contact-page">
+  <h2 class="contactme">Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <form class="formsection" action="https://formspree.io/f/xgeqgyly" method="post">
+    <input type="text" id="name" name="user_name" placeholder="Add your name" maxlength="30"/>
+    <input type="email" id="mail" name="user_email" placeholder="Add your email" maxlength="50" />
+    <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500"></textarea>     
+    <button type="submit" id="get-intouch">Get In Touch</button>
```
Comment (joskalenda) [inline index.html]: - [ ] Well done on adding the max-length as required, in addition, kindly add the `required` attribute to all the input fields, this is to activate the validation on submit.

File: style.css [hunk 2]
```diff
@@ -882,3 +882,114 @@ body {
     width: 65%;
   }
 }
+
+.contact-page {
+  display: flex;
```
Comment (joskalenda) [inline style.css:891]: - [x] Kindly make a few touches on the design to correct what is shown in these images, before the `contact me` text, consider using the same margin property as in the image to match this design.
- [x] Your text title is black, am sure you have forgotten to modify it before you submit your work, please fix that too
- [x] Take a look closer at the design, I think it is not centered at all, a big part is laying on the left side, isn't it? please fix that to make it in the center.
- [x] As indicated with the red box in the input forms, please consider making some padding, your placeholders are too much close to the side.
des1

File: style.css [hunk 3]
```diff
@@ -882,3 +882,114 @@ body {
...
+font-style: normal;
+font-weight: 700;
+font-size: 40px;
+line-height: 52px;
+
+color: #FFFFFF;
+}
+*/
+}
+
```
Comment (joskalenda) [inline style.css]: - [ ] Following best practices, it is not a good approach to leave commented lines of code, please remove this commented code here to make your code even better and clean

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @jugosack , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]...✅ Good commit messages
- [ ]...✅ Green linters
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jugosack) [reply index.html]: @joskalenda Thank you for your comment. I have fixed the code according to your recommendation.

File: style.css (hunk 2 above)
Comment (jugosack) [reply style.css:891]: @joskalenda Thank you for your comment. I have fixed the code according to your recommendation.

File: style.css (hunk 3 above)
Comment (jugosack) [reply style.css]: @joskalenda Thank you for your comment. I have fixed the code according to your recommendation.

File: index.html [hunk 4]
```diff
@@ -372,8 +372,19 @@ <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">
...
+
   </div>
- 
+ <footer class="contact-page">
+  <h2 class="contactme">Contact me</h2>
+  <p class="contact_p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <form class="formsection" action="https://formspree.io/f/xgeqgyly" method="post">
+    <input type="text" id="name" name="user_name" placeholder="Add your name" maxlength="30" required/>
+    <input type="email" id="mail" name="user_email" placeholder="Add your email" maxlength="50" required/>
+    <textarea id="msg" name="user_message" placeholder="Write your message here" maxlength="500" required></textarea>
```
Comment (ha-manel) [inline index.html:383]: - [ ] Great job with everything, you're almost there 👏 Kindly add padding to the input field as requested by the previous reviewer to match the design in Figma 💯 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @jugosack 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
- [ ] Please consider following best practices when you write commit messages, they should be capitalized sentences in the imperative form such as "Add linters" instead of "adding linters"... 👍 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (jugosack) [reply index.html:383]: @ha-manel Thank you for your comment. I have fixed the code adding padding to the input field as requested by the previous reviewer to match the design in Figma according to your recommendation.

Screen Shot 2022-12-07 at 5 34 50 PM

**Review by ichala — APPROVED** [review]
Hi @jugosack   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**